<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data | Exogenous Trips</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Data: &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Exogenous Trips&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_help">
         <property name="text">
          <string notr="true">Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTreeView" name="scenarios_tree"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Category</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cb_category"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Mode</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cb_mode"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTabWidget" name="sddd">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>Trips</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <layout class="QFormLayout" name="formLayout_3">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Origin Zone</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="cb_tr_zone_origin"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_5">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>Destination Zone</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="cb_tr_zone_destination"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_6">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Trips</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="le_trip"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QPushButton" name="add_trip">
                     <property name="text">
                      <string>Add</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="trips_tbl"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="total_items_exotrips">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>Factor</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <layout class="QFormLayout" name="formLayout_7">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Origin Zone</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="cb_fc_zone_origin"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_8">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Destination Zone</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="cb_fc_zone_destination"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_9">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>Factor</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="le_factor"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QPushButton" name="add_factor">
                     <property name="text">
                      <string>Add</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="factor_tbl"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btn_category">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_mode">
           <property name="text">
            <string>Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_help</tabstop>
  <tabstop>scenarios_tree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
